internal/daemon/worker/testing.go

```diff
@@ -177,6 +177,9 @@ type TestWorkerOpts struct {
 	// The worker credential storage KMS to use, or one will be created
 	WorkerAuthStorageKms wrapping.Wrapper
 
+	// The location of the worker's auth storage
+	WorkerAuthStoragePath string
```

Comment:
I can't figure out how this is used (if it is) anywhere?

Reply:
I think I meant to use it and then didn't end up doing so; I'll take it out for now

Reply:
Wait no it's used in the new test!